tf/service_container/provider.tf

```diff
@@ -0,0 +1,7 @@
+terraform {
+  required_providers {
+    kubernetes = {
+      source  = "hashicorp/kubernetes"
```

Comment:
Am surprised - would have thought, that this works automatically.

Reply:
It works, but you get this warning:
```
Warning: Reference to undefined provider
│ 
│   on analyzing.tf line 39, in module "mavrodi_common":
│   39:     kubernetes = kubernetes.mavrodi
│ 
│ There is no explicit declaration for local provider name "kubernetes" in module.mavrodi_common, so Terraform is assuming you mean to pass a configuration for "hashicorp/kubernetes".
│ 
│ If you also control the child module, add a required_providers entry named "kubernetes" with the source address "hashicorp/kubernetes".
```